include/bitcoin/blockchain/interface/block_chain.hpp

```diff
@@ -181,6 +181,12 @@ class BCB_API block_chain
     void fetch_block(const hash_digest& hash,
         block_fetch_handler handler) const;
 
+    void fetch_block_txs_size(const hash_digest& hash,
```

Comment:
Is it not needed the search by height in fetch_block_txs_size and...

Reply:
Right now its not necessary, since this is only used by insight.
If in the future Insight Api is updated we may need it.